CustomEd.Classroom.Service/Consumers/StudentEvents/StudentUpdatedEventConsumer.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace CustomEd.Classroom.Service.Consumers
 {
+    ///I think you forgot to replace teacher with student
```

Comment:
What do you mean?

Reply:
![Uploading err2.jpg…]()

Reply:
![err2](https://github.com/aenon0/CustomEd/assets/95585844/60a7845a-841a-4bf4-b77e-b86bbec1f7fd)